nuxeo-distribution/nuxeo-server-hotreload-tests/src/test/java/org/nuxeo/ftest/server/hotreload/ITDevHotReloadTest.java

```diff
@@ -249,4 +260,51 @@ public void testHotReloadFileImporters() throws IOException {
         assertEquals("Foo", docType);
     }
 
+    /**
+     * @since 11.1
+     */
+    @Test
+    public void shouldComputeMimeTypeWhenBlobsSchemaIsModified() throws IOException {
+        // 1st deploy that define a new document type `CustomDocType` and its model.
+        hotReloadRule.deployJarDevBundle(ITDevHotReloadTest.class, "testHotReloadSchema");
```

Comment:
Could you follow the convention of method naming: `testHotReload...`? Furthermore, this call is not needed here because you can leverage the rule behavior + you don't need to hot reload twice.